src/app/ffbe/services/characters.service.spec.ts

```diff
@@ -491,4 +496,19 @@ describe('CharactersService', () => {
     expect(character.entries['207002017'].characterEntrySkills.filter(characterEntrySkill => characterEntrySkill.id === 236047).length).toEqual(1);
   }));
 
+  it('should not load nested properties when loading a shallow character', inject([CharactersService], (service: CharactersService) => {
+    // GIVEN
+    service.loadCharactersFromDataMining();
+
+    // WHEN
+    const character: Character = service.searchForShallowCharacterByGumiId(207002017);
+
+    // THEN
+    expect(character).toBeTruthy();
+    expect(character.entries.length === 1);
+    expect(character.skills.length).toEqual(20);
+    expect(character.skills.every(skill => FfbeUtils.isNullOrUndefined(skill.skill))).toBeTrue();
```

Comment:
Ce test ne teste probablement pas grand chose. Rien ne garanti que nos mocks soient cohérents et que les skills du character `207002017` soient présents dedans. Il faut vérifier avec quelque chose qui ressemble à :
```
expect(skillService.searchForSkillByGumiId).toHaveBeenCalledTimes(0);
```
Il faut peut-être ajouter un `spyOn(skillService, 'searchForSkillByGumiId').and.callThrough();` dans le `// GIVEN` du test